nebula3/gclient/net/SessionPool.py

```diff
@@ -27,7 +27,7 @@ class SessionPool(object):
     S_OK = 0
     S_BAD = 1
 
-    def __init__(self, username, password, space_name, addresses):
+    def __init__(self, username, password, space_name, addresses, handshakeKey):
```

Comment:
do not add parameter for _init_.
the handshakeKey already defined in SessionPoolConfig.